rate_counter/templates/table_detail.html

```diff
@@ -0,0 +1,83 @@
+{% extends "base.html" %}
+
+{% block title %}
+
+    {{ table.name }} | {{ block.super }}
+
+{% endblock title %}
+
+{% block content %}
+
+<div class='col-sm-6 col-sm-offset-3' style="background: lavender">
+    <h1>{{table.name}}</h1>
+    <form method="POST" action="" enctype="multipart/form-data" >{% csrf_token %}
+        <table>
+            {% if rows %}
+                <tr>
+                    <td height="32px"><strong>Subject name</strong></td>
```

Comment:
this is called 'inline' styles, when you write css inside HTML. Bad practice. I suggest to move this into separate file with styles.